multicluster/controllers/multicluster/common/helper.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 Antrea Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import "k8s.io/apimachinery/pkg/types"
+
+const (
+	AntreaMcsLabel        = "antrea.io/multi-cluster"
+	AntreaMcsAutoGenLabel = "antrea.io/multi-cluster-autogenerated"
+	SourceImportLabel     = "antrea.io/resource-import-name"
+
+	ServiceKind       = "Service"
+	EndpointsKind     = "Endpoints"
+	ServiceImportKind = "ServiceImport"
+
+	SourceServiceType = "SourceServiceType"
+
+	DefaultWorkerCount = 5
+
+	ResourceExportFinalizer = "resourceexport.finalizers.antrea.io"
+)
+
+const (
+	Separator = '/'
+)
+
+// TODO: Use NamespacedName stringer method instead of this. eg nsName.String()
+func NamespacedName(namespace, name string) string {
+	return namespace + string(Separator) + name
+}
+
+// GetNamespacedName returns the objects name as NamespacedName struct.
+func GetNamespacedName(namespace, name string) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: namespace,
+		Name:      name,
+	}
+}
+
+func GetPointer(str string) *string {
```

Comment:
Why we need a func?

Reply:
there are a few fields requires string pointer,https://github.com/antrea-io/antrea/pull/3024/files#diff-1e1d6e2558b5bb86a952c580e877a8a072c499a0c278194afb4d0948ae459d10R362-R371, I added this to avoid define a string and get its pointer repeatedly, but looks like only serviceexporter_controller.go use this function, I will move it and name it as getPointer

Reply:
done